reconcile/utils/oc.py

```diff
@@ -591,21 +592,23 @@ def get_obj_root_owner(self, ns, obj, allow_not_found=False):
             ns (string): namespace of the object
             obj (dict): representation of the object
             allow_not_found (bool, optional): allow owner to be not found
+            allow_not_controller (bool, optional): allow non-controller owner
```

Comment:
This would be a good place to experiment with type hints. ;)

Reply:
i'm not there yet :)